### PR TITLE
Removed unnecessary usage of the importants in the style

### PR DESCRIPTION
**Pull request by dmitriy-borzenko**

### CODE REVIEW

File: src/app/dataset-create/dataset-create.component.scss [hunk 1]
```diff
@@ -151,7 +151,13 @@ hr {
...
+    .switch-kind-container {
+        .description {
+            font-size: 12px;
+        }
+    }
+
     .fs-12 {
-        font-size: 12px !important;
+        font-size: 12px;
     }
```
Comment (EvilAlexei) [inline src/app/dataset-create/dataset-create.component.scss:162]: This is a tricky one.
There is a main problem in the wrong structure of the element.
But I assume this is out of scope here to refactor a structure :)

There is already a global class fs-12, and there is no need for the same class here,
and there will be no need in the `.switch-kind-container { ... }`.

And this is a good example of where `!important` could be the right solution :)
What I mean:
 - you can add !important to the main .fs-12 class in the styles.scss
 - and remove it here inside the component, along with the `.switch-kind-container`

File: src/app/dataset-create/dataset-create.component.html [hunk 2]
```diff
@@ -30,11 +30,11 @@ <h1 class="title">Create a new dataset</h1>
             </div>
         </div>
         <mat-divider></mat-divider>
-        <div class="my-4">
+        <div class="switch-kind-container my-4">
             <label class="d-flex" for="kind-root">
                 <input name="kind" type="radio" value="root" formControlName="kind" />
```
Comment (EvilAlexei) [inline src/app/dataset-create/dataset-create.component.html:35]: Please add the `id` same as the attribute `for` on the label
```suggestion
                <input name="kind" type="radio" value="root" id="kind-root" formControlName="kind" />
```

File: src/app/dataset-create/dataset-create.component.html [hunk 3]
```diff
@@ -44,7 +44,7 @@ <h1 class="title">Create a new dataset</h1>
             <label for="kind-derivative" class="mt-2 d-flex">
                 <input name="kind" type="radio" value="derivative" formControlName="kind" />
```
Comment (EvilAlexei) [inline src/app/dataset-create/dataset-create.component.html:45]: Please add the `id` same as the attribute `for` on the label
```suggestion
                <input name="kind" type="radio" id="kind-derivative" value="derivative" formControlName="kind" />
```

File: src/app/dataset-view/additional-components/overview-component/components/edit-watermark-modal/edit-watermark-modal.component.scss [hunk 4]
```diff
@@ -1,9 +1,9 @@
 .custom-picker {
-    z-index: 9999 !important;
+    z-index: 9999;
 }
 
 .w-65 {
-    width: 67% !important;
+    width: 67%;
```
Comment (EvilAlexei) [inline src/app/dataset-view/additional-components/overview-component/components/edit-watermark-modal/edit-watermark-modal.component.scss:6]: It seems really strange :D

Class w-65 with the value 67
Could you please rename it have a garmony?

**Review by EvilAlexei — CHANGES_REQUESTED** [review]

File: src/app/auth/settings/account-settings.component.html [hunk 5]
```diff
@@ -11,7 +11,7 @@
                 class="avatar rounded-circle mr-3"
             />
             <div class="flex-auto mx-4 mt-3">
-                <div class="h3-custom lh-condensed">
+                <div>
```
[the remark this answers is not in the document]
Comment (dmitriy-borzenko) [reply src/app/auth/settings/account-settings.component.html:14]: Fixed

File: src/app/auth/settings/account-settings.component.scss [hunk 6]
```diff
@@ -1,5 +1,11 @@
 @import "var";
 
+ul {
```
[the remark this answers is not in the document]
Comment (dmitriy-borzenko) [reply src/app/auth/settings/account-settings.component.scss:3]: Fixed

File: src/app/dataset-create/dataset-create.component.scss (hunk 1 above)
Comment (dmitriy-borzenko) [reply src/app/dataset-create/dataset-create.component.scss:162]: Ok, fixed

File: src/app/dataset-create/dataset-create.component.html (hunk 2 above)
Comment (dmitriy-borzenko) [reply src/app/dataset-create/dataset-create.component.html:35]: Added

File: src/app/dataset-view/additional-components/overview-component/components/edit-watermark-modal/edit-watermark-modal.component.scss (hunk 4 above)
Comment (dmitriy-borzenko) [reply src/app/dataset-view/additional-components/overview-component/components/edit-watermark-modal/edit-watermark-modal.component.scss:6]: Fixed

**Pull request merged**